src/examples/regression_test.json

```diff
@@ -164,5 +164,18 @@
       "data": "",
       "hash": ""
     }
+  },
+  "Bitwise Primitive": {
+    "digest": "Bitwise Primitive",
+    "methods": {
+      "xor": {
+        "rows": 75,
```

Comment:
recalling that your test circuit does 1+2+3+4=10 rounds of 16-bit XOR, and it takes 60 more rows than before, id sounds like 6 additional constraints per 16-bit Xor round.

so this calculation, which is done 3 times per round, seems to take 2 rows:
```
(x - (x0 + x1 2^4 + x2 2^8 + x3 2^12)) / 2^16
```
this makes sense, because we can do 2 additions per row (double generic gate) and there are 4 additions (the multiplications by constants can be included at no additional costs)